modules/era/sql/era_droplist.sql

```diff
@@ -15412,7 +15401,7 @@ INSERT INTO `mob_droplist` VALUES (1890,2,0,1000,748,0);   -- Gold Beastcoin (St
 
 -- ZoneID: 149 - Orcish Dragoon
 INSERT INTO `mob_droplist` VALUES (1891,0,0,1000,1439,@UNCOMMON); -- Dragoons Testimony (Uncommon, 10%)
-INSERT INTO `mob_droplist` VALUES (1891,0,0,1000,1042,@RARE);     -- Davoi Coffer Key (Rare, 5%)
+INSERT INTO `mob_droplist` VALUES (1891,0,0,1000,1042,@VRARE);    -- Davoi Coffer Key (Very Rare, 1%)
```

Comment:
I am still gonna comment that all keys are probably 5%

![image](https://user-images.githubusercontent.com/105882754/189505442-8372373d-7419-44f7-9aca-8972b6eaf609.png)

It seem extremely unlikely that not all keys should have the same drop rates. With rates as low as 1% and 5% the drop data can be a little skewed. I could be completely wrong. :/